docs/course_materials/exercises/03_Variables_and_datatypes/class_exercise_1.md

```diff
@@ -1,7 +1,7 @@
 ### Let's Practice Together \#1
 
-- Create a new file **my_full_name.py** using vim editor
-- Get the user's first name and save it in a new variable **first_name**
-- Get the user's last name and save it in a new variable **last_name**
-- Print **first_name** and **last_name** using %s formatting
-- Save the file and run it on the terminal
+- Create a file *variables_class.py*
```

Comment:
wasn't this one changed by shir already?

Reply:
I just replaced the place of the files/